docs/DeveloperGuide.md

```diff
@@ -554,3 +554,77 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+
+## **Appendix: Instructions for manual testing**
+
+### Position Management
+
+#### Add a position
+
+1. Adding a position
+    1. Prerequisite: There must not be a position with `title` 'Software Tester' inside HR Manager.
+    2. Test case: `add_p title=Software Tester status=open`
+       Expected: Position Software Tester with open status is added to HR Manager. Details of the added position are shown in command feedback box.
```

Comment:
There should be a line break between Test case and the Expected. Same goes for the rest of the section

Reply:
Line breaks have been added.